packages/l10n/src/types.ts

```diff
@@ -18,3 +18,24 @@ export type TimeZone = {
   offset: string;
 };
 export type TimeZones = Record<string, TimeZone>;
+
+export type LocalizedField = {
+  locale: string;
+  value: string;
+};
+
+export type LocalizedString = {
```

Comment:
Nit: I've often seen Records being preferred here. So Record<string, string>.

Reply:
yes, that makes sense